_source/_data/shipper-tabs.yml

```diff
@@ -69,6 +69,8 @@ tags:
     name: Networking
   - slug: agents
     name: Agents
+  - slug: prometheus
+    name: Prometheus
```

Comment:
We need to change the name to cloudwatch.

Reply:
In that case, shouldn't it be 

```
- slug: aws
    name: AWS
```

Reply:
Yes we can do that, so also the logo should be aws

Reply:
Done.